src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/HtmlStyle.java

```diff
@@ -408,12 +408,30 @@
     // A module page has details tables containing the details of the directives.
 
     /**
-     * The class of a {@code table} element used to present details of a program element.
```

Comment:
This comment applies to the file as a whole, and not to this specific position in the file. (I couldn't find a way to attach a comment to the file or an unedited line.)

I'm surprised that I didn't see lots of deletions in this file.  You are removing references from the `Table` constructors, so I was expecting those table-specific names to be deleted here as well: or am I missing something?